packages/x-codemod/README.md

```diff
@@ -0,0 +1,94 @@
+# @mui/x-codemod
+
+> Codemod scripts for MUI X
+
+[![npm version](https://img.shields.io/npm/v/@mui/x-codemod.svg?style=flat-square)](https://www.npmjs.com/package/@mui/x-codemod)
+[![npm downloads](https://img.shields.io/npm/dm/@mui/x-codemod.svg?style=flat-square)](https://www.npmjs.com/package/@mui/x-codemod)
+
+This repository contains a collection of codemod scripts based for use with
+[jscodeshift](https://github.com/facebook/jscodeshift) that help update MUI X APIs.
+
+## Setup & run
+
+<!-- #default-branch-switch -->
+
+```bash
+npx @mui/x-codemod <codemod> <paths...>
+
+Applies a `@mui/x-codemod` to the specified paths
+
+Positionals:
+  codemod  The name of the codemod                                [string]
+  paths    Paths forwarded to `jscodeshift`                       [string]
+
+Options:
+  --version     Show version number                                 [boolean]
+  --help        Show help                                           [boolean]
+  --parser      which parser for jscodeshift to use.
+                                                    [string] [default: 'tsx']
+  --jscodeshift Pass options directly to jscodeshift                  [array]
+
+Examples:
+  npx @mui/x-codemod v6.0.0/preset-safe src
+  npx @mui/x-codemod v6.0.0/component-rename-prop src --
+  --component=Grid --from=prop --to=newProp
```

Comment:
Maybe use `localization-provider-rename-locale` here instead, unless you want yo show how to pass params

Reply:
This is the same example mui-codemod gives. It's a utility transform we expose to allow users making quick custom prop switching.
IMHO, it doesn't make much sense adding example of a generic transform, which will eventually end up in the migration doc anyways. 🤔

Reply:
Oh ok, then maybe change the name of the component to  `DatePicker`
I just saw `Grid` which was a bit stranged

Reply:
🤦 Damn, good spot, the fact that it was at least missing `Data` prefix slip through my eyes. 😆

Reply:
Updated it to `DataGrid` just because we now that it's a way more widely used component with way more props. 🙈